projects/social/README.md

```diff
@@ -39,22 +39,25 @@ Now that you have a graph full of users and friendships, you can crawl through t
 >>> print(connections)
 {1: [1], 8: [1, 8], 10: [1, 10], 5: [1, 5], 2: [1, 10, 2], 6: [1, 10, 6], 7: [1, 10, 2, 7]}
 ```
+
 Note that in this sample, Users 3, 4 and 9 are not in User 1's extended social network.
 
-* Hint 1: What kind of graph search guarantees you a shortest path?
-* Hint 2: Instead of using a `set` to mark users as visited, you could use a `dictionary`. Similar to sets, checking if something is in a dictionary runs in O(1) time. If the visited user is the key, what would the value be?
+-   Hint 1: What kind of graph search guarantees you a shortest path?
+-   Hint 2: Instead of using a `set` to mark users as visited, you could use a `dictionary`. Similar to sets, checking if something is in a dictionary runs in O(1) time. If the visited user is the key, what would the value be?
 
 ## 3. Questions
 
 1. To create 100 users with an average of 10 friends each, how many times would you need to call `addFriendship()`? Why?
 
-2. If you create 1000 users with an average of 5 random friends each, what percentage of other users will be in a particular user's extended social network? What is the average degree of separation between a user and those in his/her extended network?
+    - `(100 * 10) / 2 = 500`
+    - It would be called 500 times. Half as many average friend times number of users because once 1 is a friend with 2 then the possibility of 2 being friends with 1 is already taken
 
+2. If you create 1000 users with an average of 5 random friends each, what percentage of other users will be in a particular user's extended social network? What is the average degree of separation between a user and those in his/her extended network?
 
```

Comment:
average degree of separation is going to be around 5 for friends in connections, the separation is the length of one friend to the social path / length of connections, around 99% of users will be in a particular user's extended social network